src/lib/logger.rs

```diff
@@ -133,15 +134,24 @@ pub(crate) fn init(options: &LoggerOptions) {
     let result = fern::Dispatch::new()
         .format(move |out, message, record| {
             let name = env!("CARGO_PKG_NAME");
+            let level = env::var("CARGO_MAKE_INTERNAL_RECURSION")
```

Comment:
lets use the new module. and in that module, lets use envmnt::get_or("CARGO_MAKE_INTERNAL_RECURSION", "0".to_string()) which is much simpler.